behat/features/tabs.feature

```diff
@@ -0,0 +1,141 @@
+Feature: Book page
+  As an anonymous user
+  We need to be able to see the tabs list in the book page.
+
+  @api
+  Scenario: Visit a books page
+    Given I am an anonymous user
+    When  I visit "books/זה-ספר-בית-המדות"
+    Then I should see the text under the tab
```

Comment:
@bitamar @miri here's the change I've added -- https://github.com/Gizra/haskala/pull/158/files#diff-3ab04f2a2c5d5dd77b80747bfff02df7R198